drools-wb-screens/drools-wb-guided-rule-editor/drools-wb-guided-rule-editor-client/src/main/java/org/drools/workbench/screens/guided/rule/client/editor/RuleModeller.java

```diff
@@ -118,6 +122,7 @@ public RuleModeller(final RuleModel model,
                         final boolean isReadOnly) {
         this(model,
              Collections.emptyList(),
+             Collections.emptyList(),
```

Comment:
You should check the operation of "Guided Rule Templates" - as without the validators being passed here - you could probably use an OptaPlanner reserved binding in "Guided Rule Templates"... I guess you don't care for DRL files re-using OptaPlanner bindings?